alpine/content.md

```diff
@@ -0,0 +1,30 @@
+# What is Alpine Linux?
+
+[Alpine Linux](http://alpinelinux.org/) is a Linux distribution built around [musl libc](http://www.musl-libc.org/) and [BusyBox](http://www.busybox.net/). The image is only 5 MB in size and has access to a [package repository](http://forum.alpinelinux.org/packages) that is much more complete than other BusyBox based images. This makes Alpine Linux a great image base for utilities and even production applications. [Read more about Alpine Linux here](https://www.alpinelinux.org/about/) and you can see how their mantra fits in right at home with Docker images.
+
+# How to use this image
+
+## Usage
+
+Stop doing this:
+
+	FROM ubuntu:14.04
+	RUN apt-get update -q \
+	  && DEBIAN_FRONTEND=noninteractive apt-get install -qy mysql-client \
+	  && apt-get clean \
+	  && rm -rf /var/lib/apt
+	ENTRYPOINT ["mysql"]
+
+This took 19 seconds to build and yields a 164 MB image. Eww.
```

Comment:
Yea, same here. It is closer to 232 MB. Maybe I just did my test wrong when I originally wrote that...